parm/config/gfs/config.resources

```diff
@@ -734,6 +734,7 @@ case ${step} in
         exit 4
       ;;
     esac
+    if [[ ${machine} == "JET" ]]; then export memory_upp="0" ; fi
```

Comment:
What happens if it get "some" memory?

Reply:
As in the case of esfc, xjet and kjet have very different size of memory per nodes. Setting 0 (to use all available memory) made sense to keep the modifications to minimal and allows higher resolution to run on kjet while lower res can run on xjet.

Reply:
would `unset memory_upp` work instead of `export memory_upp=0`?

Reply:
I don't know if `unset memory_upp` would work or not. I have not used that option before. Let me tag @DavidHuber-NOAA to see if he tried that before.

Reply:
@aerorahul I changed to `unset memory_upp` for jet